src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -27,8 +27,8 @@ private void VisitFieldType(IFieldSymbol symbol)
 
         public override void VisitField(IFieldSymbol symbol)
         {
-            AddAccessibilityIfRequired(symbol);
-            AddMemberModifiersIfRequired(symbol);
+            AddAccessibilityIfNeeded(symbol);
+            AddMemberModifiersIfNeeded(symbol);
             AddFieldModifiersIfRequired(symbol);
```

Comment:
Consider renaming `AddFieldModifiersIfRequired` too for consistency

Reply:
I would say methods in this area suffixed with 'IfRequired' generally :)

Reply:
I'm curious about this too. Why rename two of the three APIs here?

Reply:
Because I derped.